Content/Serializer/MediaSerializer.php

```diff
@@ -80,21 +80,26 @@ public function serialize(MediaInterface $media, string $locale, ?SerializationC
         unset($mediaData['adminUrl']);
         unset($mediaData['_hash']);
 
+        $formatMediaApi = $apiMedia;
+        if($media->getPreviewImage()) {
+            $formatMediaApi = new Media($media->getPreviewImage(), $locale);
```

Comment:
Looks like there is currently a false typehint in the sulu core here so we can workaround this currently with:

```suggestion
        /** @var Media|null $previewImage */
        $previewImage = $media->getPreviewImage();
        if($previewImage) {
            $formatMediaApi = new Media($previewImage, $locale);
```